packages/util/src/errors.ts

```diff
@@ -66,7 +66,8 @@ export interface StringLike {
 }
 
 export interface ErrorData {
-  [key: string]: StringLike | undefined;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  [key: string]: any;
```

Comment:
Can it be `unknown`?

Reply:
Yes, but to get around an error in line 151 I had to change `.toString()` to `String()`, hopefully that's ok.